Sources/EliminationMenu.swift

```diff
@@ -98,23 +98,23 @@ public class EliminationMenu: UIView {
         
         if menu.isLeftAligned {
             view.addConstraint(NSLayoutConstraint(
-                item: menu, attribute: .Left, relatedBy: .Equal, toItem: view, attribute: .Left, multiplier: 1, constant: margin.x)
+                item: view, attribute: .LeftMargin, relatedBy: .Equal, toItem: menu, attribute: .Left, multiplier: 1, constant: margin.x)
             )
         }
         else {
             view.addConstraint(NSLayoutConstraint(
-                item: view, attribute: .Right, relatedBy: .Equal, toItem: menu, attribute: .Right, multiplier: 1, constant: margin.x)
+                item: view, attribute: .RightMargin, relatedBy: .Equal, toItem: menu, attribute: .Right, multiplier: 1, constant: margin.x)
```

Comment:
I pinned the edges to the view on purpose to [provide compatibility to iOS 7](https://github.com/r-dent/EliminationMenu/commit/16f6f638e8e39c3f8080ac80416ee9b82131405c). Thought that would be a good Idea :-)